### ISSUE TITLE
New version?

### ISSUE THREAD

**Issue by kloczek** [storaged-project/blivet-gui#342]
https://github.com/storaged-project/blivet-gui/compare/2.3.0-1...master shows +160 commits added since last version.
What about new version? 😃

**Comment by vojtechtrefny** [comment]
That's mostly test and translations, I have some new features planned for a new release, but that will take more time (and blivet 3.5.0 needs to be released first which will also take some time). I might do a new micro release with some important bug fixes cherry picked from master-

**Comment by kloczek** [comment]
BTW translations.
I have problem with updating thsoe files
```console
[tkloczko@devel-g2v blivet-gui-2.3.0]$ make
make -C po
make[1]: Entering directory '/home/tkloczko/rpmbuild/BUILD/blivet-gui-2.3.0/po'
../translation-canary/xgettext_werror.sh --default-domain=blivet-gui --add-comments --keyword=_ --keyword=N_ --keyword=P_:1,2 ../blivetgui/actions_menu.py ../blivetgui/actions_toolbar.py ../blivetgui/blivetgui.py ../blivetgui/blivet_utils.py ../blivetgui/config.py ../blivetgui/exception_handler.py ../blivetgui/gui_utils.py ../blivetgui/i18n.py ../blivetgui/__init__.py ../blivetgui/list_actions.py ../blivetgui/list_devices.py ../blivetgui/list_parents.py ../blivetgui/list_partitions.py ../blivetgui/loading_window.py ../blivetgui/logs.py ../blivetgui/main_menu.py ../blivetgui/osinstall.py ../blivetgui/processing_window.py ../blivetgui/communication/client.py ../blivetgui/communication/constants.py ../blivetgui/communication/errors.py ../blivetgui/communication/__init__.py ../blivetgui/communication/proxy_utils.py ../blivetgui/communication/server.py ../blivetgui/dialogs/add_dialog.py ../blivetgui/dialogs/constants.py ../blivetgui/dialogs/device_info_dialog.py ../blivetgui/dialogs/edit_dialog.py ../blivetgui/dialogs/helpers.py ../blivetgui/dialogs/__init__.py ../blivetgui/dialogs/message_dialogs.py ../blivetgui/dialogs/other_dialogs.py ../blivetgui/dialogs/size_chooser.py ../blivetgui/dialogs/widgets.py ../blivetgui/visualization/__init__.py ../blivetgui/visualization/logical_view.py ../blivetgui/visualization/physical_view.py ../blivetgui/visualization/rectangle.py ../data/ui/about_dialog.ui ../data/ui/add_disklabel_dialog.ui ../data/ui/blivet-gui.ui ../data/ui/cache_area.ui ../data/ui/confirm_actions_dialog.ui ../data/ui/confirm_delete_dialog.ui ../data/ui/confirm_dialog.ui ../data/ui/custom_dialog.ui ../data/ui/encryption_chooser.ui ../data/ui/error_dialog.ui ../data/ui/exception_dialog.ui ../data/ui/format_dialog.ui ../data/ui/info_dialog.ui ../data/ui/label_dialog.ui ../data/ui/luks_passphrase_dialog.ui ../data/ui/mountpoint_dialog.ui ../data/ui/parent_area.ui ../data/ui/parent_chooser.ui ../data/ui/raid_chooser.ui ../data/ui/resize_dialog.ui ../data/ui/root_check_window.ui ../data/ui/show_actions_dialog.ui ../data/ui/size_area.ui ../data/ui/size_chooser.ui ../data/ui/unmount_dialog.ui ../data/ui/warning_dialog.ui --from-code utf-8
xgettext: warning: file '../data/ui/about_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/add_disklabel_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/blivet-gui.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/cache_area.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/confirm_actions_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/confirm_delete_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/confirm_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/custom_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/encryption_chooser.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/error_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/exception_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/format_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/info_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/label_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/luks_passphrase_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/mountpoint_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/parent_area.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/parent_chooser.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/raid_chooser.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/resize_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/root_check_window.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/show_actions_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/size_area.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/size_chooser.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/unmount_dialog.ui' extension 'ui' is unknown; will try C
xgettext: warning: file '../data/ui/warning_dialog.ui' extension 'ui' is unknown; will try C
make[1]: *** [Makefile:34: potfile] Error 1
make[1]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/blivet-gui-2.3.0/po'
make: *** [Makefile:13: all] Error 2
```

**Comment by vojtechtrefny** [comment]
> BTW translations.
> I have problem with updating thsoe files

Interesting, what version of gettext are you using? We used to use intltool to extract strings from the Glade files, but that was removed [years ago](https://github.com/storaged-project/blivet-gui/commit/d0a6336353f50994c32f6b5e07e3e9f27fdc90f2) and since then gettext seems to be able to extract them without issues. It might be an issue with different gettext configuration maybe? I'll try to find out more.
